Tools/TestWebKitAPI/TestWebKitAPI.xcodeproj/project.pbxproj

```diff
@@ -1335,6 +1387,16 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		3A15783D28D1505B00142DB1 /* Product Dependencies */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 16;
+			files = (
```

Comment:
I think you need to add WebKit.framework to product build dependencies, as is added below.

The TestWebKitAPI/config.h uses headers generated by WebKit.framework build.

Reply:
OK. I wasn't using anything from `WebKit.framework` so I removed it.

Reply:
Yeah, anything that build on TestWebKitAPI needs to build _after_ WebKit. Otherwise, a build of the whole stack could fail trying to build TestWGSL before `WebKit.framework` has any content in it.